doc/source/multivisit.rst

```diff
@@ -3,4 +3,435 @@
+IR mosaics are also generated at a 'coarse' plate scale of 0.12"/pixel to minimize the resampling artifacts while also being easily
+scaled to the 'fine' plate scale mosaics.
+
+SkyCell Example
+'''''''''''''''
+For example, observations have been taken of NGC5474 with both the ACS and WFC3 cameras.  The ACS observations were taken
+with the ACS/WFC detector using the F814W and F606W filters, while the WFC3 observations were taken using the IR detector
+using the F110W and F160W filters.  All these observations fall within the SkyCell at position X06y11 in the
+projection cell p2381, but given the dramatic plate scale differences, these observations can not be used to create a
+single mosaic.  Instead, 6 separate layers get defined for this SkyCell; namely,
+
+  * acs_wfc_f814w  (0.04"/pixel)
+  * acs_wfc_f606w  (0.04"/pixel)
+  * wfc3_ir_f110w  (0.04"/pixel)
+  * wfc3_ir_f160w  (0.04"/pixel)
+  * wfc3_ir_f110w_coarse  (0.12"/pixel)
+  * wfc3_ir_f160w_coarse  (0.12"/pixel)
+
+Since they all have the same WCS, modulo the plate scale differences, they can be overlaid pixel-by-pixel with each other for
+analysis.
+
+
+MVM Processing Steps
+====================
+The definitions for the ProjectionCell and SkyCell allow for all HST observations to be processed into a logical set of
+image mosaic products regardless of how many observations cover any particular spot on the sky while tying them all
+together in the same astrometric reference frame (as much as possible, anyway).  The steps taken to generate these
+MVM products can be summarized as:
+
+  * Determine what SkyCell or set of SkyCells each exposure overlaps
+  * Copy all relevant exposures for a given SkyCell into a single directory for that SkyCell
+  * Rename input exposures to have MVM-specific filenames
+  * Generate input file to be used for processing each SkyCell
+  * Evaluate all input exposures to define all layers needed for the SkyCell
+  * Determine which layer to process
+  * Drizzle all exposures for each layer to be processed to create new mosaic product for that layer
+
+SkyCell Membership
+------------------
+Data from HST gets orgranized based on a filename derived from the proposal used to define the observations and how
+HST should take them.  MVM processing, on the other hand, focuses on how the observations relate to each other on the
+sky based on the WCS information.  The `drizzlepac/haputils/cell_utils.py` module includes the code used to interpret
+the WCS information for exposures and determine what ProjectionCells and SkyCells each exposure overlaps as shown in the
+section on the `Code for Defining SkyCell ID`_.
+
+This code can be called on any set of user-defined exposures to either determine for the first time what SkyCells the
+exposures overlap.  During HST pipeline processing, this code gets called for the exposures from each visit once they
```

Comment:
The "During HST pipeline processing" sentence is missing some words or ???

Reply:
Changed in 241a672